### PR TITLE
add time series example notebook

### PR DESCRIPTION
**Pull request by zicanl-amazon**
*Issue #, if available:*

*Description of changes: Add examples for TimeSeries Explainability

*Testing done: Notebook runs end to end

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: sagemaker-clarify/time_series_byom/time_series_bring_your_own_model.ipynb [hunk 1]
```diff
@@ -0,0 +1,1244 @@
...
+    "- `num_samples`: Number of samples to be used in the Asymmetric Shapley Value forecasting algorithm. Only applicable when using `\"fine_grained\"`  explanations. This represents the number of permutations sampled for computing the ASV. \n",
+    "- `baseline`: baseline configuration (dictionary). The baseline config is used to replace out-of-coalition values for the corresponding datasets (also known as background data). For temporal data (target time series, related time series), the baseline value types are `\"zero\"`, where all out-of-coalition values will be replaced with `0.0`, or `\"mean\"`, all out-of-coalition values will be replaced with the average of a time series. For static data(static covariates), a baseline value for each covariate should be provided for each possible item_id. An example config follows, where ``item1`` and ``item2`` are item ids::\n",
```
Comment (rvasahu-amazon) [inline sagemaker-clarify/time_series_byom/time_series_bring_your_own_model.ipynb:700]: Formatting for baseline isn't working here, can you format it as a code block? Just copy paste this in.

\`\`\` 
{
&emsp;"target_time_series": "zero",
&emsp;"related_time_series": "zero",
&emsp;"static_covariates":
&emsp;&emsp;"item1": [1, 1],
&emsp;&emsp;"item2": [0, 1],
&emsp;}
}
\`\`\`

Comment (rvasahu-amazon) [reply sagemaker-clarify/time_series_byom/time_series_bring_your_own_model.ipynb:700]: Make sure to do for both notebooks.

**Review by aws-byeldos — APPROVED** [review]
I skimmed through it. LGTM!

I approve assuming there are only minor fixes of the previous comments and you did not add any considerable changes since the last several times I reviewed this change.

Please, be specific if you added anything that we have not discussed (if that slipped through)

File: sagemaker-clarify/time_series_byom/time_series_bring_your_own_model.ipynb (hunk 1 above)
Comment (zicanl-amazon) [reply sagemaker-clarify/time_series_byom/time_series_bring_your_own_model.ipynb:706]: addressed in the next revision

**Review by rvasahu-amazon — APPROVED** [review]
Going to approve this.

Ideally speaking I would like a couple other changes such as:

- Modifying the pretty print calls at the end of both notebooks to make the output more readable (by changing spacing parameters).
- Adding a table of contents to the BYOM notebook like the one in the DeepAR notebook (and making sure they're consistent with the sections actually present in the notebooks)
- Some grammar fixes.

However, in the interest of your time, perhaps these things are something you or I can work on at a later date.

**Review by mufaddal-rohawala — APPROVED** [review]

**Pull request merged**